static/src/js/routes/SavedProjects/__tests__/SavedProjects.test.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import Enzyme, { shallow } from 'enzyme'
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
+import Helmet from 'react-helmet'
+
+import * as AppConfig from '../../../../../../sharedUtils/config'
+
+import SavedProjects from '../SavedProjects'
+import SavedProjectsContainer
+  from '../../../containers/SavedProjectsContainer/SavedProjectsContainer'
+
+// Mock react-leaflet because it causes errors
+jest.mock('react-leaflet', () => ({
+  createLayerComponent: jest.fn().mockImplementation(() => {}),
+  createControlComponent: jest.fn().mockImplementation(() => {})
+}))
+
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
If we can avoid it we should use RTL on new tests so we don't accumulate new tech debt

Reply:
Great point! I converted all of these! Thank you for keeping me honest!